src/expr/term_conversion_proof_generator.cpp

```diff
@@ -18,27 +18,57 @@ using namespace CVC4::kind;
 
 namespace CVC4 {
 
+std::ostream& operator<<(std::ostream& out, TConvPolicy tcpol)
+{
+  switch (tcpol)
+  {
+    case TConvPolicy::FIXPOINT: out << "FIXPOINT"; break;
+    case TConvPolicy::ONCE: out << "ONCE"; break;
+    default: out << "TConvPolicy:unknown"; break;
```

Comment:
```suggestion
    default: out << "TConvPolicy:unknown"; 
```

Reply:
This would be a compile error?

Reply:
That's strange. The `default` case does not need a break.